ahbottomnavigation/src/main/java/com/aurelhubert/ahbottomnavigation/notification/AHNotification.java

```diff
@@ -0,0 +1,117 @@
+package com.aurelhubert.ahbottomnavigation.notification;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.ColorInt;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author repitch
+ */
+public class AHNotification implements Parcelable {
+
+    @Nullable
+    private String text; // can be null, so notification will not be shown
+
+    @ColorInt
+    private int textColor; // if 0 then use default value
+
+    @ColorInt
+    private int backgroundColor; // if 0 then use default value
+
+    private AHNotification() {
+        // empty
+    }
+
+    private AHNotification(Parcel in) {
+        text = in.readString();
+        textColor = in.readInt();
+        backgroundColor = in.readInt();
+    }
+
+    public boolean isEmpty() {
+        return TextUtils.isEmpty(text);
+    }
+
+    public String getText() {
+        return text;
+    }
+
+    public int getTextColor() {
+        return textColor;
+    }
+
+    public int getBackgroundColor() {
+        return backgroundColor;
+    }
+
+    public static AHNotification justText(String text) {
+        return newBuilder().setText(text).build();
+    }
+
+    public static Builder newBuilder() {
```

Comment:
fixed in f9acf56
I removed this method, use `new AHNotification.Builder()` instead